docs/docs/add-seo-component.md

```diff
@@ -218,7 +218,7 @@ const query = graphql`
 
 You could also put the Facebook and Twitter meta-tags into their own components, add custom favicons you placed in your `static` folder, and add [schema.org](https://schema.org/) data (Google will use that for their [Structured Data](https://developers.google.com/search/docs/guides/intro-structured-data)). To see how that works you can have a look at these two examples:
 
-- [marisamorby.com](https://github.com/jlengstorf/marisamorby.com/blob/master/src/components/SEO/SEO.js)
+- [marisamorby.com](https://github.com/jlengstorf/marisamorby.com/blob/master/packages/gatsby-theme-blog-sanity/src/components/seo.js)
```

Comment:
I suggest going for the original url instead of a fork
```suggestion
- [marisamorby.com](https://github.com/marisamorby/marisamorby.com/blob/master/packages/gatsby-theme-blog-sanity/src/components/seo.js)
```